### ISSUE TITLE
Fertilizer does nothing 

### ISSUE THREAD

**Issue by Nanders** [StrangeLoopGames/EcoIssues#4125]
(from steam comment)

Fertilizers are broken at the moment.
I used each and every one of them of tiled soil and they didn't affect numbers (nitrogen etc.) in any way.

**Comment by LiaungYip** [comment]
This bug means that crop yield in a plot of land will fall due to lack of nutrients, as the soil nutrients can't be replaced. This makes it difficult to maintain food supply over time.

**Comment by ownyHD** [comment]
https://github.com/StrangeLoopGames/EcoIssues/labels/High%20Priority

**Comment by ownyHD** [comment]
i would like to add high priority label to this. how do i do that?

**Comment by LiaungYip** [comment]
@ownyHD : The Eco developers add the labels according to their judgement of what's most important. It's not for players to decide the priority of each issue!

(That said, I do agree that this issue is pretty severe - see screenshot:)

![image](https://user-images.githubusercontent.com/2173684/36070313-a5e6c56c-0f43-11e8-935e-9df942f1d23f.png)

**Comment by Rushyo** [comment]
Been doing some debugging on my test server. It looks like the fertiliser values being added, it's just returning 0 when queried.

Technical translation: AccumulationPuller's registeredChanges are being updated correctly, but running Apply() straight afterwards always returns 0.0. I wonder if registeredChanges (a ConcurrentDictionary) is locked, as:

```cs  
      if (!this.registeredChanges.TryGetValue(area, out num))
        return 0.0f;
    }````

**Comment by knexer** [comment]
Hey all, just letting you know we've fixed this issue. The fix will be released shortly - part of 7.1.0 I believe.
